avsomr/avsomr.cpp

```diff
@@ -21,12 +21,11 @@ AvsOmr::AvsOmr()
 
 void AvsOmr::resolve()
       {
-
       auto firstMS = [](Sheet* sh) -> const MStack* {
             if (sh->page.systems.empty())
                   return nullptr;
 
-            for (size_t si = 0; si < sh->page.systems.size(); ++si) {
+            for (int si = 0; si < sh->page.systems.size(); ++si) {
```

Comment:
why? `size()` returns an unsigned value (usually)

Reply:
But check the compiler warnings, esp. those from MSVC (like on AppVeyor, if you're not using MSVC yourself)

Reply:
Replaced to remove warning.
`size()` for std types returns `size_t`, but the `size()` of Qt types returns `int`

Reply:
Replaced `size()' on 'count()' so that there is less confusion

Reply:
Oh, OK. in that case you're perfectly right. Althouigh IMHO this is a bug in Qt.

Reply:
Hmm, tricky, changing it to `count()`, guess I'm gonna "borrow" that idea...